rpm/qtwebsockets.spec

```diff
@@ -50,21 +47,19 @@ rm -f %{buildroot}%{_libdir}/lib*.la
 
 %files
 %defattr(-,root,root,755)
-%doc *GPL*
+%license LICENSE.LGPLv21 LGPL_EXCEPTION.txt LICENSE.LGPLv3
```

Comment:
I'm fairly sure this should include also LICENSE.GPLv3 as LGPLv3 is an extension of that.

Reply:
Added here too.